--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/solenv/inc/_tg_shl.mk	Sat Sep 29 03:11:33 2001
+++ solenv/inc/_tg_shl.mk	Thu Nov 22 08:12:04 2001
@@ -139,7 +139,12 @@
 .IF "$(UNIXVERSIONNAMES)"!=""
 .IF "$(OS)"!="MACOSX"
 .IF "$(GUI)"=="UNX"
+.IF "$(OS)"=="IRIX"
+# IRIX doesn't like the "'s
+SHL1SONAME=$(SONAME_SWITCH)$(SHL1TARGETN:b:b)
+.ELSE
 SHL1SONAME=\"$(SONAME_SWITCH)$(SHL1TARGETN:b:b)\"
+.ENDIF
 .ENDIF			# "$(GUI)"!="UNX"
 .ENDIF			# "$(OS)"!="MACOSX"
 .ENDIF			# "$(UNIXVERSIONNAMES)"!=""
@@ -570,7 +575,12 @@
 .IF "$(UNIXVERSIONNAMES)"!=""
 .IF "$(OS)"!="MACOSX"
 .IF "$(GUI)"=="UNX"
+.IF "$(OS)"=="IRIX"
+# IRIX doesn't like the "'s
+SHL2SONAME=$(SONAME_SWITCH)$(SHL2TARGETN:b:b)
+.ELSE
 SHL2SONAME=\"$(SONAME_SWITCH)$(SHL2TARGETN:b:b)\"
+.ENDIF
 .ENDIF			# "$(GUI)"!="UNX"
 .ENDIF			# "$(OS)"!="MACOSX"
 .ENDIF			# "$(UNIXVERSIONNAMES)"!=""
@@ -1001,7 +1011,12 @@
 .IF "$(UNIXVERSIONNAMES)"!=""
 .IF "$(OS)"!="MACOSX"
 .IF "$(GUI)"=="UNX"
+.IF "$(OS)"=="IRIX"
+# IRIX doesn't like the "'s
+SHL3SONAME=$(SONAME_SWITCH)$(SHL3TARGETN:b:b)
+.ELSE
 SHL3SONAME=\"$(SONAME_SWITCH)$(SHL3TARGETN:b:b)\"
+.ENDIF
 .ENDIF			# "$(GUI)"!="UNX"
 .ENDIF			# "$(OS)"!="MACOSX"
 .ENDIF			# "$(UNIXVERSIONNAMES)"!=""
@@ -1432,7 +1447,12 @@
 .IF "$(UNIXVERSIONNAMES)"!=""
 .IF "$(OS)"!="MACOSX"
 .IF "$(GUI)"=="UNX"
+.IF "$(OS)"=="IRIX"
+# IRIX doesn't like the "'s
+SHL4SONAME=$(SONAME_SWITCH)$(SHL4TARGETN:b:b)
+.ELSE
 SHL4SONAME=\"$(SONAME_SWITCH)$(SHL4TARGETN:b:b)\"
+.ENDIF
 .ENDIF			# "$(GUI)"!="UNX"
 .ENDIF			# "$(OS)"!="MACOSX"
 .ENDIF			# "$(UNIXVERSIONNAMES)"!=""
@@ -1863,7 +1883,12 @@
 .IF "$(UNIXVERSIONNAMES)"!=""
 .IF "$(OS)"!="MACOSX"
 .IF "$(GUI)"=="UNX"
+.IF "$(OS)"=="IRIX"
+# IRIX doesn't like the "'s
+SHL5SONAME=$(SONAME_SWITCH)$(SHL5TARGETN:b:b)
+.ELSE
 SHL5SONAME=\"$(SONAME_SWITCH)$(SHL5TARGETN:b:b)\"
+.ENDIF
 .ENDIF			# "$(GUI)"!="UNX"
 .ENDIF			# "$(OS)"!="MACOSX"
 .ENDIF			# "$(UNIXVERSIONNAMES)"!=""
@@ -2294,7 +2319,12 @@
 .IF "$(UNIXVERSIONNAMES)"!=""
 .IF "$(OS)"!="MACOSX"
 .IF "$(GUI)"=="UNX"
+.IF "$(OS)"=="IRIX"
+# IRIX doesn't like the "'s
+SHL6SONAME=$(SONAME_SWITCH)$(SHL6TARGETN:b:b)
+.ELSE
 SHL6SONAME=\"$(SONAME_SWITCH)$(SHL6TARGETN:b:b)\"
+.ENDIF
 .ENDIF			# "$(GUI)"!="UNX"
 .ENDIF			# "$(OS)"!="MACOSX"
 .ENDIF			# "$(UNIXVERSIONNAMES)"!=""
@@ -2725,7 +2755,12 @@
 .IF "$(UNIXVERSIONNAMES)"!=""
 .IF "$(OS)"!="MACOSX"
 .IF "$(GUI)"=="UNX"
+.IF "$(OS)"=="IRIX"
+# IRIX doesn't like the "'s
+SHL7SONAME=$(SONAME_SWITCH)$(SHL7TARGETN:b:b)
+.ELSE
 SHL7SONAME=\"$(SONAME_SWITCH)$(SHL7TARGETN:b:b)\"
+.ENDIF
 .ENDIF			# "$(GUI)"!="UNX"
 .ENDIF			# "$(OS)"!="MACOSX"
 .ENDIF			# "$(UNIXVERSIONNAMES)"!=""
@@ -3156,7 +3191,12 @@
 .IF "$(UNIXVERSIONNAMES)"!=""
 .IF "$(OS)"!="MACOSX"
 .IF "$(GUI)"=="UNX"
+.IF "$(OS)"=="IRIX"
+# IRIX doesn't like the "'s
+SHL8SONAME=$(SONAME_SWITCH)$(SHL8TARGETN:b:b)
+.ELSE
 SHL8SONAME=\"$(SONAME_SWITCH)$(SHL8TARGETN:b:b)\"
+.ENDIF
 .ENDIF			# "$(GUI)"!="UNX"
 .ENDIF			# "$(OS)"!="MACOSX"
 .ENDIF			# "$(UNIXVERSIONNAMES)"!=""
@@ -3587,7 +3627,12 @@
 .IF "$(UNIXVERSIONNAMES)"!=""
 .IF "$(OS)"!="MACOSX"
 .IF "$(GUI)"=="UNX"
+.IF "$(OS)"=="IRIX"
+# IRIX doesn't like the "'s
+SHL9SONAME=$(SONAME_SWITCH)$(SHL9TARGETN:b:b)
+.ELSE
 SHL9SONAME=\"$(SONAME_SWITCH)$(SHL9TARGETN:b:b)\"
+.ENDIF
 .ENDIF			# "$(GUI)"!="UNX"
 .ENDIF			# "$(OS)"!="MACOSX"
 .ENDIF			# "$(UNIXVERSIONNAMES)"!=""
@@ -4018,7 +4063,12 @@
 .IF "$(UNIXVERSIONNAMES)"!=""
 .IF "$(OS)"!="MACOSX"
 .IF "$(GUI)"=="UNX"
+.IF "$(OS)"=="IRIX"
+# IRIX doesn't like the "'s
+SHL10SONAME=$(SONAME_SWITCH)$(SHL10TARGETN:b:b)
+.ELSE
 SHL10SONAME=\"$(SONAME_SWITCH)$(SHL10TARGETN:b:b)\"
+.ENDIF
 .ENDIF			# "$(GUI)"!="UNX"
 .ENDIF			# "$(OS)"!="MACOSX"
 .ENDIF			# "$(UNIXVERSIONNAMES)"!=""
