--- /projects/sise/openoffice/devel/workpits/OO641B_STLP45/source/i18npool/prj/build.lst	Fri Nov 16 21:19:19 2001
+++ i18npool/prj/build.lst	Tue Dec  4 09:14:34 2001
@@ -1,4 +1,4 @@
-inp  i18npool    :   bridges sax stoc NULL
+inp  i18npool    :   bridges sax stoc comphelper NULL
 inp  i18npool                                    usr1    -   all in_mkout NULL
 inp  i18npool\source\unotypes                    nmake   -   all in_utypes NULL
 inp  i18npool\source\registerservices            nmake   -   all in_rserv in_utypes NULL
