--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/rsc/source//rscpp/cpp3.c	Wed Apr 18 20:31:56 2001
+++ rsc/source//rscpp/cpp3.c	Fri Nov 16 09:42:54 2001
@@ -79,7 +79,11 @@
 
 #if defined(DEBUG) && (HOST == SYS_VMS || HOST == SYS_UNIX)
 #include        <signal.h>
+#ifdef IRIX
+extern void 	abort(void);
+#else
 extern int      abort();                /* For debugging                */
+#endif
 #endif
 
 void InitCpp3()
