--- /projects/sise/openoffice/devel/workpits/OO641B_STLP45/source/solenv/bin/checkdll.sh	Tue Jun 19 23:07:46 2001
+++ solenv/bin/checkdll.sh	Wed Dec 12 16:34:08 2001
@@ -35,6 +35,16 @@
 	    *) DYLD_LIBRARY_PATH=$libpath;;
         esac
         export DYLD_LIBRARY_PATH;;
+    IRIX) case "${LD_LIBRARYN32_PATH+X}" in
+	    X) LD_LIBRARYN32_PATH=$libpath:$LD_LIBRARYN32_PATH;;
+	    *) LD_LIBRARYN32_PATH=$libpath;;
+	esac
+        export LD_LIBRARYN32_PATH;;
+    IRIX64) case "${LD_LIBRARYN32_PATH+X}" in
+	    X) LD_LIBRARYN32_PATH=$libpath:$LD_LIBRARYN32_PATH;;
+	    *) LD_LIBRARYN32_PATH=$libpath;;
+	esac
+        export LD_LIBRARYN32_PATH;;
 	*)  case "${LD_LIBRARY_PATH+X}" in
 	    X) LD_LIBRARY_PATH=$libpath:$LD_LIBRARY_PATH;;
 	    *) LD_LIBRARY_PATH=$libpath;;
