--- /projects/sise/openoffice/devel/workpits/OO641B_STLP45/workarea/cppu/inc/com/sun/star/uno/Reference.hxx	Mon Jan  7 11:21:50 2002
+++ cppu/inc/com/sun/star/uno/Reference.hxx	Thu Jan 10 10:03:32 2002
@@ -71,6 +71,8 @@
 #include <com/sun/star/uno/genfunc.hxx>
 #endif
 
+#include <stdio.h>
+
 /** */ //for docpp
 namespace com
 {     
@@ -148,6 +150,7 @@
 template< class interface_type >
 inline Reference< interface_type >::Reference( const Reference< interface_type > & rRef ) SAL_THROW( () )
 {
+	printf(" \b");
 	_pInterface = rRef._pInterface;
 	if (_pInterface)
 		_pInterface->acquire();
@@ -156,6 +159,7 @@
 template< class interface_type >
 inline Reference< interface_type >::Reference( interface_type * pInterface ) SAL_THROW( () )
 {
+	printf(" \b");
 	_pInterface = pInterface;
 	if (_pInterface)
 		_pInterface->acquire();
@@ -182,6 +186,7 @@
 template< class interface_type >
 inline Reference< interface_type >::Reference( XInterface * pInterface, __UnoReference_Query ) SAL_THROW( (RuntimeException) )
 {
+	printf(" \b");
 	_pInterface = __query( pInterface );
 }
 //__________________________________________________________________________________________________
