--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/dbaccess/source/ui/querydesign/TableConnection.cxx	Mon Oct  8 17:32:34 2001
+++ dbaccess/source/ui/querydesign/TableConnection.cxx	Tue Nov 13 10:21:55 2001
@@ -58,6 +58,9 @@
  *
  *
  ************************************************************************/
+
+#include <algorithm>
+
 #ifndef DBAUI_TABLECONNECTION_HXX
 #include "TableConnection.hxx"
 #endif
