--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/dbaccess/source/ui/querydesign/JoinTableView.cxx	Mon Oct 22 18:09:10 2001
+++ dbaccess/source/ui/querydesign/JoinTableView.cxx	Fri Nov  9 08:15:13 2001
@@ -58,6 +58,9 @@
  *
  *
  ************************************************************************/
+
+#include <algorithm>
+
 #ifndef DBAUI_QUERYTABLEVIEW_HXX
 #include "JoinTableView.hxx"
 #endif
