--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/dbaccess/source/ui/browser/genericcontroller.cxx	Thu Oct 18 16:47:58 2001
+++ dbaccess/source/ui/browser/genericcontroller.cxx	Fri Nov  9 07:29:55 2001
@@ -58,6 +58,9 @@
  *
  *
  ************************************************************************/
+
+#include <algorithm>
+
 #ifndef DBAUI_GENERICCONTROLLER_HXX
 #include "genericcontroller.hxx"
 #endif
