--- /projects/sise/openoffice/devel/workpits/OO641B_STLP45/source/moz/prj/d.lst	Tue Oct 16 20:58:26 2001
+++ moz/prj/d.lst	Wed Dec  5 09:43:23 2001
@@ -2,9 +2,9 @@
 mkdir: %_DEST%\inc%_EXT%\mozilla\nspr
 mkdir: %_DEST%\inc%_EXT%\mozilla\nspr\obsolete
 
-..\%__SRC%\inc\* %_DEST%\inc%_EXT%\mozilla
-..\%__SRC%\inc\nspr\* %_DEST%\inc%_EXT%\mozilla\nspr
-..\%__SRC%\inc\nspr\obsolete\* %_DEST%\inc%_EXT%\mozilla\nspr\obsolete
-..\%__SRC%\lib\* %_DEST%\lib%_EXT%
-..\%__SRC%\bin\*.zip %_DEST%\bin%_EXT%
+..\%__SRC%\inc\mozilla\* %_DEST%\inc%_EXT%\mozilla\*
+..\%__SRC%\inc\mozilla\nspr\* %_DEST%\inc%_EXT%\mozilla\nspr\*
+..\%__SRC%\inc\mozilla\nspr\obsolete\* %_DEST%\inc%_EXT%\mozilla\nspr\obsolete\*
+..\%__SRC%\lib\* %_DEST%\lib%_EXT%\*
+..\%__SRC%\bin\*.zip %_DEST%\bin%_EXT%\*.zip
 
