--- /projects/sise/openoffice/devel/workpits/OO641B_STLP45/source/product/prj/d.lst	Thu Jun 28 00:07:30 2001
+++ product/prj/d.lst	Tue Dec  4 10:37:06 2001
@@ -12,6 +12,7 @@
 ..\inc\scp\udk_completefiles.scp %_DEST%\inc%_EXT%\scp\udk_completefiles.scp
 
 ..\%__SRC%\lib\lib*static*.dylib %_DEST%\lib%_EXT%\lib*static*.dylib
+..\%__SRC%\lib\*.so %_DEST%\lib%_EXT%\*.so
 ..\%__SRC%\misc\*staticdatamembers.cxx %_DEST%\inc%_EXT%\*staticdatamembers.cxx
 ..\%__SRC%\misc\*staticdatamembers.h* %_DEST%\inc%_EXT%\*staticdatamembers.h*
 
