--- /projects/sise/openoffice/devel/workpits/source/OpenOffice641/product/util/makefile.mk	Thu Oct 11 20:10:42 2001
+++ product/util/makefile.mk	Wed Mar  6 15:49:05 2002
@@ -77,6 +77,9 @@
 STLPORTLIBNAME=libstlport_sunpro.so
 .ENDIF
 
+ADDITIONAL_DLLS= \
+	$(DESTDIRDLL)$/$(MY_DLLPREFIX)$(BINDINGDLL)$(MY_DLLPOSTFIX)
+
 .ELIF "$(OS)$(CPU)"=="SOLARISI"
 ##########
 # SOLARIS INTEL
@@ -91,6 +94,24 @@
 BINDINGDLL=sunpro5_uno
 .ENDIF
 
+ADDITIONAL_DLLS= \
+	$(DESTDIRDLL)$/$(MY_DLLPREFIX)$(BINDINGDLL)$(MY_DLLPOSTFIX) 
+
+.ELIF "$(OS)"=="IRIX"
+###########
+# IRIX
+###########
+DESTDIRBIN=$(DESTDIR)$/irix$/bin
+DESTDIRLIB=$(DESTDIR)$/irix$/lib
+DESTDIRDLL=$(DESTDIRLIB)
+.IF "$(COM)"=="GCC"
+#BINDINGDLL=gcc2_uno
+STLPORTLIBNAME=libstlport_gcc.so
+.ELSE
+BINDINGDLL=mips_uno
+STLPORTLIBNAME=libstlport_mips.so
+.ENDIF
+
 .ELIF "$(OS)"=="LINUX"
 ###########
 # LINUX
@@ -101,6 +122,9 @@
 STLPORTLIBNAME=libstlport_gcc.so
 BINDINGDLL=gcc2_uno
 
+ADDITIONAL_DLLS= \
+	$(DESTDIRDLL)$/$(MY_DLLPREFIX)$(BINDINGDLL)$(MY_DLLPOSTFIX) 
+ 
 .ELIF "$(OS)"=="WNT"
 ###############
 # WINDOWS
@@ -110,7 +134,8 @@
 BINDINGDLL=msci_uno
 ADDITIONAL_DLLS= \
 	$(DESTDIRDLL)$/stlport_vc6.dll \
-	$(DESTDIRDLL)$/w9xucwrp.dll
+	$(DESTDIRDLL)$/w9xucwrp.dll \
+	$(DESTDIRDLL)$/$(MY_DLLPREFIX)$(BINDINGDLL)$(MY_DLLPOSTFIX)
 .ENDIF
 
 .IF "$(STLPORTLIBNAME)" != ""
@@ -231,7 +256,6 @@
 	$(DESTDIRDLL)$/$(MY_DLLPREFIX)salhelper$(UDK_MAJOR)$(COM)$(MY_DLLPOSTFIX)	\
 	$(DESTDIRDLL)$/$(MY_DLLPREFIX)javaloader$(MY_DLLPOSTFIX)          		\
 	$(DESTDIRDLL)$/$(MY_DLLPREFIX)juh$(MY_DLLPOSTFIX)                   		\
-	$(DESTDIRDLL)$/$(MY_DLLPREFIX)$(BINDINGDLL)$(MY_DLLPOSTFIX)           		\
 	$(DESTDIRDLL)$/$(MY_DLLPREFIX)java_uno$(MY_DLLPOSTFIX)            		\
 	$(DESTDIRDLL)$/$(MY_DLLPREFIX)prot_uno_uno$(MY_DLLPOSTFIX)            		\
 	$(DESTDIRDLL)$/$(MY_DLLPREFIX)iiop_uno$(MY_DLLPOSTFIX)                		\
@@ -360,8 +384,8 @@
 	$(GNUCOPY) -p $(DLLOUT)$/$(@:f) $@
 
 $(DESTDIRDLL)$/%$(MY_VERSION_LINKPOSTFIX) : $(DESTDIRDLL)$/%$(MY_VERSION_DLLPOSTFIX) $(DIRLIST)
-	+ln -s $(@:f)$(MY_VERSION_LINKTARGETPOSTFIX) $@
-	+ln -s $(@:f)$(MY_VERSION_LINKTARGETPOSTFIX) $(@:db)
+	+ln -sf $(@:f)$(MY_VERSION_LINKTARGETPOSTFIX) $@
+	+ln -sf $(@:f)$(MY_VERSION_LINKTARGETPOSTFIX) $(@:db)
 
 $(DESTDIRJAR)$/%.jar : $(BINOUT)$/%.jar $(DIRLIST)
 	$(GNUCOPY) -p $(BINOUT)$/$(@:f) $@
