--- config_office/configure.in	2002/04/17 20:37:42	1.41.2.2
+++ config_office/configure.in	2002/06/26 04:50:08
@@ -283,7 +283,7 @@
          AC_MSG_WARN([found egcs- version,  use (stable=gcc-) version of the compiler]) >> warn
          _gcc_version=`$GCC --version | $AWK -F"-" '{ print \$2 }'`
       else
-         _gcc_version=`$GCC --version`
+         _gcc_version=`$GCC -dumpversion`
       fi
       _gcc_major=`echo $_gcc_version | $AWK -F. '{ print \$1 }'` 
       if test -n "$enable_gcc3"; then 
@@ -976,6 +976,7 @@
 dnl testing for X libraries and includes...
 dnl ***************************************
 if test "$_os" = "Darwin" -a "x$x_includes" = "xno_x_includes"; then
+   echo "Do Nothing for _os = Darwin and x_includes = no_x_includes"
    dnl Mac OS X using Aqua graphics. Don't check for X11.
 elif test "$_os" != "WINNT" ; then
    AC_PATH_X
