--- /projects/sise/openoffice/devel/workpits/OOo641C/workarea/svx/util/makefile.mk	Tue Oct  2 22:48:58 2001
+++ svx/util/makefile.mk	Fri Feb 15 15:19:21 2002
@@ -102,6 +102,7 @@
 SHL1BASE  = 0x1d800000
 SHL1STDLIBS= \
             $(SALLIB) \
+            $(INETLIBSH) \
             $(VOSLIB) \
             $(TOOLSLIB) \
             $(SVTOOLLIB) \
@@ -113,7 +114,6 @@
             $(SFX2LIB) \
             $(GOODIESLIB) \
             $(BASICLIB) \
-            $(INETLIBSH) \
             $(CHANELLIB) \
             $(SVMEMLIB) \
             $(TKLIB) \
