--- /projects/sise/openoffice/devel/workpits/OOo641C/workarea/sch/source/ui/view/schdrgmt.cxx	2001/11/07 15:59:19	1.2
+++ sch/source/ui/view/schdrgmt.cxx	2002/01/09 20:42:47
@@ -135,7 +135,7 @@
 {
 	if( DragStat().CheckMinMoved( rPnt ) )
 	{
-		Vector2D aShiftVector( Vector2D( SAL_STATIC_CAST( Pair, rPnt )) - aTranslationVector );
+		Vector2D aShiftVector( (Vector2D( SAL_STATIC_CAST( Pair, rPnt )) - aTranslationVector) );
 		double fShift = aConstraintVector.Scalar( aShiftVector ) / fConstraintVecScalar;	// projection
 
 		if( fShift )
