--- /projects/sise/openoffice/devel/workpits/OOo641C/workarea/product/prj/d.lst	Thu Jun 28 00:07:30 2001
+++ product/prj/d.lst	Fri Jan 18 16:28:36 2002
@@ -15,5 +15,7 @@
 ..\%__SRC%\misc\*staticdatamembers.cxx %_DEST%\inc%_EXT%\*staticdatamembers.cxx
 ..\%__SRC%\misc\*staticdatamembers.h* %_DEST%\inc%_EXT%\*staticdatamembers.h*
 
+..\%__SRC%\lib\libremoteclientsample.so %_DEST%\lib%_EXT%\libremoteclientsample.so
+
 dos: sh -c "if test %OS% = MACOSX; then create-bundle %_DEST%\lib%_EXT%\*.dylib; fi"
 dos: sh -c "if test %OS% = MACOSX; then create-libstatic-link %_DEST%\lib%_EXT%; fi"
