--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/vcl/prj/build.lst	Fri Oct 12 00:38:19 2001
+++ vcl/prj/build.lst	Fri Nov  2 13:39:33 2001
@@ -1,4 +1,4 @@
-vc	vcl	:	nas freetype psprint rsc sot ucbhelper unotools sysui NULL
+vc	vcl	:	freetype psprint rsc sot ucbhelper unotools sysui NULL
 vc	vcl										usr1	-	all	vc_mkout NULL
 vc	vcl\source\unotypes						nmake	-	all	vc_unot NULL
 vc	vcl\source\glyphs						nmake	-	all	vc_glyphs NULL
