--- /projects/sise/openoffice/devel/workpits/source/OpenOffice641/config_office/configure.in	Tue Nov 13 08:45:53 2001
+++ config_office/configure.in	Mon Dec  3 10:41:01 2001
@@ -686,8 +686,12 @@
 				AC_MSG_ERROR([STLport4 libraries not found])
 			fi
 		elif test "$_os" = "IRIX" -o "$_os" = "IRIX64"; then
-			if test -f "$STLPORT4/lib/libstlport_mipspro_41.so"; then
-				AC_MSG_RESULT([checked])
+			if test "$CC" = "cc"; then
+				if test -f "$STLPORT4/lib/libstlport_mipspro_41.so"; then
+					AC_MSG_RESULT([checked])
+				else
+					AC_MSG_ERROR([STLport4 libraries not found])
+				fi
 			else
 				if test -f "$STLPORT4/lib/libstlport_gcc.so"; then
 					AC_MSG_RESULT([checked])
