--- /projects/sise/openoffice/devel/workpits/641-gcc-debug/source/dbaccess/source/ui/querydesign/TableWindowTitle.cxx	Thu Oct 11 18:38:13 2001
+++ dbaccess/source/ui/querydesign/TableWindowTitle.cxx	Mon Nov 26 07:05:51 2001
@@ -104,6 +104,8 @@
 #include "JoinController.hxx"
 #endif
 
+#include <algorithm>
+
 using namespace dbaui;
 using namespace ::com::sun::star::beans;
 using namespace ::com::sun::star::uno;
