--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/dbaccess/source/ui/misc/WExtendPages.cxx	Mon Jul  2 23:21:57 2001
+++ dbaccess/source/ui/misc/WExtendPages.cxx	Tue Nov 13 13:11:15 2001
@@ -58,6 +58,9 @@
  *
  *
  ************************************************************************/
+
+#include <numeric>
+
 #ifndef DBAUI_WIZ_EXTENDPAGES_HXX
 #include "WExtendPages.hxx"
 #endif
