--- /projects/sise/openoffice/devel/workpits/OO641B_STLP45/source/berkeleydb/prj/d.lst	Thu Jun 14 22:37:48 2001
+++ berkeleydb/prj/d.lst	Thu Dec  6 09:57:25 2001
@@ -8,8 +8,11 @@
 ..\%__SRC%\lib\libdb_java32.lib %_DEST%\lib%_EXT%\libdb_java32.lib
 
 ..\%__SRC%\lib\libdb-3.2.so %_DEST%\lib%_EXT%\libdb-3.2.so
+..\%__SRC%\lib\libdb-3.2.so %_DEST%\lib%_EXT%\libdb-3.2.so.
 ..\%__SRC%\lib\libdb_java-3.2.so %_DEST%\lib%_EXT%\libdb_java-3.2.so
+..\%__SRC%\lib\libdb_java-3.2.so %_DEST%\lib%_EXT%\libdb_java-3.2.so.
 ..\%__SRC%\lib\libdb_cxx-3.2.so %_DEST%\lib%_EXT%\libdb_cxx-3.2.so
+..\%__SRC%\lib\libdb_cxx-3.2.so %_DEST%\lib%_EXT%\libdb_cxx-3.2.so.
 
 ..\%__SRC%\bin\libdb32.dll %_DEST%\bin%_EXT%\libdb32.dll
 ..\%__SRC%\bin\libdb_java32.dll %_DEST%\bin%_EXT%\libdb_java32.dll
