--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/berkeleydb/makefile.mk	Thu Jun 21 04:43:52 2001
+++ berkeleydb/makefile.mk	Thu Nov 22 10:39:06 2001
@@ -83,7 +83,11 @@
 CONFIGURE_FLAGS=--enable-cxx --enable-java --enable-dynamic --enable-shared
 
 BUILD_DIR=$(CONFIGURE_DIR)
+.IF "$(OS)"=="IRIX"
+BUILD_ACTION=gmake
+.ELSE
 BUILD_ACTION=make
+.ENDIF
 
 OUT2LIB=$(BUILD_DIR)$/.libs$/libdb*.so
 
