--- /projects/sise/openoffice/devel/workpits/source/OO641/extensions/source/bibliography/formcontrolcontainer.cxx	Mon Oct 22 17:32:53 2001
+++ extensions/source/bibliography/formcontrolcontainer.cxx	Fri Nov 16 14:48:28 2001
@@ -59,6 +59,8 @@
  *
  ************************************************************************/
 
+#include <algorithm>
+
 #ifndef EXTENSIONS_BIB_FORMCONTROLCONTAINER_HXX
 #include "formcontrolcontainer.hxx"
 #endif
