--- /projects/sise/openoffice/devel/workpits/OO641B_STLP45/source/product/examples/cpp/remoteclient/makefile.mk	Thu Jul 26 18:18:04 2001
+++ product/examples/cpp/remoteclient/makefile.mk	Thu Dec 13 10:09:37 2001
@@ -67,6 +67,8 @@
 ENABLE_EXCEPTIONS=TRUE
 NO_BSYMBOLIC=TRUE
 COMP1TYPELIST=$(TARGET)
+TARGETTYPE=CUI
+NO_DEFAULT_STL=TRUE
 
 # --- Settings -----------------------------------------------------
 
@@ -127,6 +129,7 @@
 
 $(MISC)$/component_remoteclient_registered.flag .SETDIR=$(CURRENTDIR): 
 	-rm -f ..$/misc$/component_remoteclient_registered.flag
+	-cp $(SHL1TARGETN:f) $(SOLARVERSION)$/$(INPATH)$/lib$/$(SHL1TARGETN:f)
 	regmerge $(PATHPREFIX)$/server.rdb / $(SOLARBINDIR)$/udkapi.rdb 
 	regmerge $(PATHPREFIX)$/client.rdb / $(SOLARBINDIR)$/udkapi.rdb
 	regcomp -register -r $(PATHPREFIX)$/client.rdb -c $(DLLPREFIX)remoteclientsample$(DLLPOSTFIX) 
