--- /projects/sise/openoffice/devel/workpits/OOo641C/workarea/sal/rtl/source/strtmpl.c	Fri Jan 25 21:31:28 2002
+++ sal/rtl/source/strtmpl.c	Fri Jan 25 22:20:57 2002
@@ -1104,9 +1104,11 @@
         }
         else if ( *pStr == '+' )
         {
-            if ( bExp && (nExp != 0) )
+            if ( bExp ){
+			  if ( nExp != 0 )
                 break;
-            if ( fRet != 0.0 )
+			}
+            else if ( fRet != 0.0 )
                 break;
         }
         /* Exponent separator */
