--- /projects/sise/openoffice/devel/workpits/source/OO641C/setup2/script/setup.sh	Sat Nov 24 02:46:33 2001
+++ setup2/script/setup.sh	Wed Mar 13 16:13:01 2002
@@ -125,6 +125,14 @@
 	 LD_LIBRARY_PATH=./local:.:..:$sd_archive_path:$LD_LIBRARY_PATH
 	 export LD_LIBRARY_PATH
 	 ;;
+  IRIX)
+	 LD_LIBRARYN32_PATH=./local:.:..:$sd_archive_path:$LD_LIBRARYN32_PATH
+	 export LD_LIBRARYN32_PATH
+	 ;;
+  IRIX64)
+	 LD_LIBRARYN32_PATH=./local:.:..:$sd_archive_path:$LD_LIBRARYN32_PATH
+	 export LD_LIBRARYN32_PATH
+	 ;;
       *)
 	 ;;
 esac
