--- /projects/sise/openoffice/devel/workpits/source/OO641C/idlc/source/makefile.mk	Wed Apr 18 21:04:25 2001
+++ idlc/source/makefile.mk	Wed Mar 13 15:54:14 2002
@@ -148,6 +148,6 @@
 
 $(MISC)$/parser.cxx: parser.yy
 	+bison -v -d -o$(MISC)$/parser.cxx parser.yy
-	+$(COPY) $(MISC)$/parser.cxx.h $(OUT)$/inc$/parser.h
+	+$(COPY) $(MISC)$/parser.hxx $(OUT)$/inc$/parser.h
 # with line statements (for debugging)
 #	+bison -v -d -o$(MISC)$/parser.cxx parser.yy
