--- /projects/sise/openoffice/devel/workpits/641-gcc-debug/source/config_office/configure.in	Wed Sep  5 04:06:24 2001
+++ config_office/configure.in	Thu Nov 22 17:48:52 2001
@@ -731,9 +731,18 @@
    _correct_jdk_exists="false"
    dnl java -version sends output to stderr!
    _jdk=`$JAVA -J-version 2>&1 | $AWK -F'"' '{ print \$2 }'`
-   if test "$_os" = "SunOS" -o "$_os" = "IRIX" -o "$_os" = "IRIX64" -o "$_os" = "OSF1"; then
+   if test "$_os" = "SunOS" -o "$_os" = "OSF1"; then
+      _jdk_middle=`echo $_jdk | $AWK -F. '{ if ($2 != 2) print "false"; else print "true" }'`
+   fi
+   if test "$_os" = "IRIX" -o "$_os" = "IRIX64"; then
       _jdk_middle=`echo $_jdk | $AWK -F. '{ if ($2 != 2) print "false"; else print "true" }'`
+      _jdk_middle3=`echo $_jdk | $AWK -F. '{ if ($2 != 3) print "false"; else print "true" }'`
+		     
+      if test "$_jdk_middle" = "false" -a "$_jdk_middle3" = "true"; then
+         _jdk_minor="true"
+      fi      
    fi
+
    if test "$_os" = "Linux" -a "$_machine_type" = "i686"; then
       _jdk_middle=`echo $_jdk | $AWK -F. '{ if ($2 != 2) print "false"; else print "true" }'`
       _jdk_middle3=`echo $_jdk | $AWK -F. '{ if ($2 != 3) print "false"; else print "true" }'`
