--- /projects/sise/openoffice/devel/workpits/641-gcc-debug/source/sch/source/filter/xml/SchXMLWrapper.cxx	Thu Oct 18 23:06:16 2001
+++ sch/source/filter/xml/SchXMLWrapper.cxx	Wed Nov 28 18:35:33 2001
@@ -226,7 +226,7 @@
 		// convert rtl::OUString => tools String => ByteString
 		String aStr( aEx.Message );
 		ByteString aBStr( aStr, RTL_TEXTENCODING_ASCII_US );
-		ByteString aBStrStreamName( String( rsStreamName ), RTL_TEXTENCODING_ASCII_US );
+		ByteString aBStrStreamName( String( RTL_CONSTASCII_USTRINGPARAM(rsStreamName) ), RTL_TEXTENCODING_ASCII_US );
 		DBG_ERROR2( "SAX parse exception caught during import of \"%s\" stream(): %s",
 					aBStrStreamName.GetBuffer(),
 					aBStr.GetBuffer());
@@ -242,7 +242,7 @@
 		// convert rtl::OUString => tools String => ByteString
 		String aStr( aEx.Message );
 		ByteString aBStr( aStr, RTL_TEXTENCODING_ASCII_US );
-		ByteString aBStrStreamName( String( rsStreamName ), RTL_TEXTENCODING_ASCII_US );
+		ByteString aBStrStreamName( String( RTL_CONSTASCII_USTRINGPARAM(rsStreamName) ), RTL_TEXTENCODING_ASCII_US );
 		DBG_ERROR2( "SAX exception caught during import of \"%s\" stream(): %s",
 					aBStrStreamName.GetBuffer(),
 					aBStr.GetBuffer());
@@ -255,7 +255,7 @@
 		// convert rtl::OUString => tools String => ByteString
 		String aStr( aEx.Message );
 		ByteString aBStr( aStr, RTL_TEXTENCODING_ASCII_US );
-		ByteString aBStrStreamName( String( rsStreamName ), RTL_TEXTENCODING_ASCII_US );
+		ByteString aBStrStreamName( String( RTL_CONSTASCII_USTRINGPARAM(rsStreamName) ), RTL_TEXTENCODING_ASCII_US );
 		DBG_ERROR2( "IO exception caught during import of \"%s\" stream(): %s",
 					aBStrStreamName.GetBuffer(),
 					aBStr.GetBuffer());
@@ -268,7 +268,7 @@
 		// convert rtl::OUString => tools String => ByteString
 		String aStr( aEx.Message );
 		ByteString aBStr( aStr, RTL_TEXTENCODING_ASCII_US );
-		ByteString aBStrStreamName( String( rsStreamName ), RTL_TEXTENCODING_ASCII_US );
+		ByteString aBStrStreamName( String( RTL_CONSTASCII_USTRINGPARAM(rsStreamName) ), RTL_TEXTENCODING_ASCII_US );
 		DBG_ERROR2( "UNO exception caught during import of \"%s\" stream(): %s",
 					aBStrStreamName.GetBuffer(),
 					aBStr.GetBuffer());
@@ -412,7 +412,7 @@
 		// convert rtl::OUString => tools String => ByteString
 		String aStr( aEx.Message );
 		ByteString aBStr( aStr, RTL_TEXTENCODING_ASCII_US );
-		ByteString aBStrStreamName( String( rsStreamName ), RTL_TEXTENCODING_ASCII_US );
+		ByteString aBStrStreamName( String( RTL_CONSTASCII_USTRINGPARAM( rsStreamName ) ), RTL_TEXTENCODING_ASCII_US );
 		DBG_ERROR2( "Exception caught during Export of \"%s\" stream(): %s",
 					aBStrStreamName.GetBuffer(),
 					aBStr.GetBuffer());
