--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/svtools/source/contnr/makefile.mk	Mon May 21 21:13:43 2001
+++ svtools/source/contnr/makefile.mk	Mon Nov 12 14:12:53 2001
@@ -77,6 +77,11 @@
 
 # --- Files --------------------------------------------------------
 
+.IF "$(OS)"=="IRIX"
+# gcc compiler bug causes a crash hee
+CFLAGS+= -g
+.ENDIF
+
 .IF "$(COM)"=="WTC"
 CFLAGS=$(CFLAGS) -od -3r
 .ENDIF
