--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/svtools/source/config/viewoptions.cxx	Fri Oct 19 18:16:30 2001
+++ svtools/source/config/viewoptions.cxx	Thu Nov  8 12:00:14 2001
@@ -66,6 +66,7 @@
 //_________________________________________________________________________________________________________________
 
 #include "viewoptions.hxx"
+#include <stdio.h>
 
 #ifndef _COM_SUN_STAR_UNO_ANY_HXX_
 #include <com/sun/star/uno/Any.hxx>
