--- /projects/sise/openoffice/devel/workpits/641-gcc-debug/source/vcl/unx/inc/saldata.hxx	Sat Feb 10 01:46:35 2001
+++ vcl/unx/inc/saldata.hxx	Thu Dec 20 07:32:47 2001
@@ -94,7 +94,7 @@
 
 DECLARE_LIST( SalDisplays, SalDisplay* )
 
-#if defined SCO || defined LINUX || defined NETBSD || defined AIX || defined HPUX || defined FREEBSD
+#if defined SCO || defined LINUX || defined NETBSD || defined AIX || defined HPUX || defined FREEBSD || defined IRIX
 #include <pthread.h>
 #else
 typedef	unsigned int pthread_t;
