--- /projects/sise/openoffice/devel/workpits/641-gcc-debug/source/offmgr/util/makefile.mk	Wed Aug 15 19:25:03 2001
+++ offmgr/util/makefile.mk	Mon Nov 26 08:30:54 2001
@@ -122,37 +122,16 @@
 SHL2LIBS=\
 		$(SLB)$/$(TARGET).lib
 
-SHL2STDLIBS=$(TOOLSLIB) \
-			$(SVTOOLLIB) \
-			$(SVLLIB)	\
-			$(SVLIB) \
-			$(SO2LIB) \
-			$(SVMEMLIB) \
-			$(SOTLIB) \
-			$(VOSLIB) \
-			$(CHANELLIB) \
-			$(SFX2LIB) \
-			$(GOODIESLIB) \
-			$(BASICLIB) \
-			$(INETLIBSH) \
-			$(SVXLIB) \
-			$(TKLIB) \
-			$(ONELIB) \
-			$(SALLIB) \
-			$(CPPULIB) \
-			$(CPPUHELPERLIB) \
-			$(UNOTOOLSLIB) \
-            $(COMPHELPERLIB)
 
 .IF "$(GUI)" == "MAC"
-SHL2STDLIBS+= \
+SHL2STDLIBS= \
 			$(SOLARBINDIR)$/SDB$(UPD)$(DLLPOSTFIX).DLL \
 			$(SOLARLIBDIR)$/plugctor.lib \
 			$(SOLARLIBDIR)$/ysch.lib \
                         $(SOLARLIBDIR)$/ysm.lib \
                         $(SOLARLIBDIR)$/ybctl.lib
 .ELSE
-SHL2STDLIBS+=           $(BASICIDELIB)
+SHL2STDLIBS=           $(BASICIDELIB)
 .ENDIF			# "$(GUI)" == "MAC"
 
 .IF "$(GUI)" == "UNX"
@@ -166,6 +145,28 @@
 		$(LIBPRE) advapi32.lib
 .ENDIF
 
+SHL2STDLIBS+= 		$(SVLIB) \
+			$(SO2LIB) \
+			$(SVMEMLIB) \
+			$(SOTLIB) \
+			$(CHANELLIB) \
+			$(SFX2LIB) \
+			$(GOODIESLIB) \
+			$(BASICLIB) \
+			$(INETLIBSH) \
+			$(VOSLIB) \
+			$(SVXLIB) \
+			$(TKLIB) \
+			$(ONELIB) \
+			$(SALLIB) \
+			$(CPPULIB) \
+			$(CPPUHELPERLIB) \
+			$(UNOTOOLSLIB) \
+            		$(COMPHELPERLIB) \
+			$(SVTOOLLIB) \
+			$(SVLLIB)	\
+	    		$(TOOLSLIB)
+
 #.IF "$(GUI)" != "UNX"
 #SHL2OBJS=	\
 #			$(SLO)$/app.obj
