--- /projects/sise/openoffice/devel/workpits/source/OO641B/connectivity/prj/build.lst	Tue Nov 13 09:21:41 2001
+++ connectivity/prj/build.lst	Thu Mar  7 10:54:16 2002
@@ -1,4 +1,4 @@
-cn	connectivity	:	comphelper moz svtools NULL
+cn	connectivity	:	comphelper svtools NULL
 cn	connectivity							usr1	-	all	cn_mkout NULL
 cn	connectivity\source\types				nmake	-	all	cn_types NULL
 cn	connectivity\source\commontools			nmake	-	all	cn_cmtools cn_types NULL
@@ -7,8 +7,6 @@
 cn	connectivity\source\resource			nmake	-	all	cn_res NULL
 cn	connectivity\source\sdbcx				nmake	-	all	cn_sdbcx cn_types NULL
 cn	connectivity\source\drivers\ado			nmake	-	w	cn_ado cn_dbtools NULL
-cn	connectivity\source\drivers\mozab\mozillasrc		nmake	-	all	cn_mozab_mozillasrc cn_file NULL
-cn	connectivity\source\drivers\mozab	nmake	-	all	cn_mozab cn_mozab_mozillasrc cn_dbtools NULL
 cn	connectivity\source\drivers\calc		nmake	-	all	cn_calc cn_file NULL
 cn	connectivity\source\drivers\odbc		nmake	-	all	cn_odbc cn_dbtools NULL
 cn	connectivity\source\drivers\jdbc		nmake	-	all	cn_jdbc cn_dbtools NULL
