--- /projects/sise/openoffice/devel/workpits/OOo641C/workarea/sch/source/ui/unoidl/ChXDiagram.cxx	Wed Oct 10 19:26:32 2001
+++ sch/source/ui/unoidl/ChXDiagram.cxx	Wed Nov 28 17:53:44 2001
@@ -1516,7 +1516,7 @@
                         RTL_CONSTASCII_USTRINGPARAM (" is not known in :ChXDiagram::getPropertyValues"));
 				}
 #ifdef DBG_UTIL
-				ByteString sName (String(*pPropertyName), RTL_TEXTENCODING_ASCII_US);
+				ByteString sName (String(RTL_CONSTASCII_USTRINGPARAM(*pPropertyName)), RTL_TEXTENCODING_ASCII_US);
 				DBG_ERROR2 ("Diagram: Property %s has an invalid ID (%d)",
 					sName.GetBuffer(), nWID);
 #endif
