--- /projects/sise/openoffice/devel/workpits/source/OO641C/shell/prj/build.lst	Fri Nov 16 22:21:29 2001
+++ shell/prj/build.lst	Thu Mar 14 10:25:26 2002
@@ -1,4 +1,4 @@
-sh	shell	: offapi rdbmaker NULL
+sh	shell	: offuh rdbmaker NULL
 sh  shell\source\win32\					nmake	-	w sh_win32 NULL
 sh	shell\source\win32\simplemail		nmake	-	w sh_win32_simplemail NULL
 sh  shell\source\unix\exec				nmake	-	u sh_win32 NULL
