--- /projects/sise/openoffice/devel/workpits/OOo641C/workarea/tools/prj/build.lst	Mon Jan  7 08:16:15 2002
+++ tools/prj/build.lst	Mon Jan 21 15:54:47 2002
@@ -1,4 +1,4 @@
-tl	tools	:	cppu offapi external vos NULL
+tl	tools	:	cppu offuh external vos NULL
 tl	tools								usr1	-	all	tl_mkout NULL
 tl	tools\bootstrp\isdll				get		-	all	tl_bsisdll NULL
 tl	tools\bootstrp\addexes				get		-	all	tl_bsexes NULL
