--- file_contexts.orig	2005-06-17 07:53:59.000000000 -0400
+++ file_contexts	2005-06-17 07:55:36.000000000 -0400
@@ -304,6 +304,8 @@
 /opt/.*/sbin(/.*)?		system_u:object_r:sbin_t
 /opt/.*/man(/.*)?		system_u:object_r:man_t
 /opt/.*/var/lib(64)?(/.*)?		system_u:object_r:var_lib_t
+# for Openoffice.org
+/opt/.*/lib(.*)?\.so(\.[^/]*)*       --      system_u:object_r:shlib_t
 
 #
 # /etc
@@ -410,6 +412,8 @@
 /usr/local/src(/.*)?		system_u:object_r:src_t
 /usr/local/man(/.*)?		system_u:object_r:man_t
 /usr/local/.*\.so(\.[^/]*)*	--	system_u:object_r:shlib_t
+# for openoffice 1.1.x
+/usr/local/OpenOffice(.*)?/lib(.*)?\.so(\.[^/]*)*       -- system_u:object_r:shlib_t
 
 #
 # /usr/X11R6/man
