--- /projects/sise/openoffice/devel/workpits/OOO_STABLE/source/sal/osl/unx/module.c	Mon Sep  3 21:08:37 2001
+++ sal/osl/unx/module.c	Fri Nov  2 09:43:52 2001
@@ -79,6 +79,32 @@
 #include <osl/file.h>
 #endif
 
+#ifdef IRIX
+#ifndef _RLD_INTERFACE_DLFCN_H_DLADDR
+#define _RLD_INTERFACE_DLFCN_H_DLADDR
+typedef struct DL_INFO {
+	const char * dli_fname;
+	void       * dli_fbase;
+	const char * dli_sname;
+	void       * dli_saddr;
+	int          dli_version;
+	int          dli_reserved1;
+	long         dli_reserved[4];
+} Dl_info;
+#endif
+#include <rld_interface.h>
+#define _RLD_DLADDR             14
+int dladdr(void *address, Dl_info *dl);
+
+int dladdr(void *address, Dl_info *dl)
+{
+	void *v;
+	v = _rld_new_interface(_RLD_DLADDR,address,dl);
+
+	return (int)v;
+}
+#endif
+
 #include "system.h"
 
 #if defined(DEBUG)
