--- vcl/workben/officeacceptthread.cxx	2001/09/11 15:52:22	1.2
+++ vcl/workben/officeacceptthread.cxx	2002/01/11 00:32:57
@@ -122,13 +122,13 @@
 			sal_Bool static isProcessRunning( ::vos::OProcess::TProcessInfo& aProcessInfo )
 			{
 				int nRet;
-#if defined (SOLARIS)
+#if defined (SOLARIS) || defined(IRIX)
 				int status;
 				nRet = waitpid(aProcessInfo.Ident, &status,WNOHANG);
 #elif defined(LINUX)
 				nRet = kill(aProcessInfo.Ident, 0);
 #endif
-#if defined (UNX)
+#if defined (LINUX) || defined(IRIX)  || defined (SOLARIS)
 				if ( nRet == 0 )
 					return sal_True; // process is running
 				if ( nRet < 0 )
