--- /projects/sise/openoffice/devel/workpits/source/OO641/desktop/source/offwrp/makefile.mk	Thu Sep 27 08:02:16 2001
+++ desktop/source/offwrp/makefile.mk	Wed Oct 31 09:45:55 2001
@@ -83,6 +83,12 @@
 
 SHL1IMPLIB= iwrp
 SHL1STDLIBS=    \
+					$(SCHLIB)               \
+					$(SMLIB)                \
+					$(SWLIB)                \
+					$(SDLIB)                \
+					$(SCLIB)                \
+					$(SVXLIB)               \
 				$(SFX2LIB) 			\
 				$(FWELIB)			\
 				$(BASICLIB) 		\
@@ -102,12 +108,6 @@
 				$(UNOLIB)			\
 				$(CPPULIB)			\
 				$(CPPUHELPERLIB)	\
-					$(SCHLIB)               \
-					$(SMLIB)                \
-					$(SWLIB)                \
-					$(SDLIB)                \
-					$(SCLIB)                \
-					$(SVXLIB)               \
 				$(UNOTOOLSLIB)						\
 				$(COMPHELPERLIB)
 
