--- /projects/sise/openoffice/devel/workpits/OOo641C/workarea/vcl/unx/source/app/i18n_ic.cxx	2001/11/15 12:35:44	1.19
+++ vcl/unx/source/app/i18n_ic.cxx	2002/01/11 02:49:53
@@ -356,7 +356,7 @@
 
         if ( mnPreeditStyle != XIMPreeditNone )
         {
-#if defined LINUX || defined FREEBSD || defined NETBSD
+#if defined LINUX || defined FREEBSD || defined NETBSD || defined IRIX
             if ( mpPreeditAttributes != NULL )
 #endif
                 mpAttributes = XVaAddToNestedList( mpAttributes,
@@ -364,7 +364,7 @@
         }
         if ( mnStatusStyle != XIMStatusNone )
         {
-#if defined LINUX || defined FREEBSD || defined NETBSD
+#if defined LINUX || defined FREEBSD || defined NETBSD || defined IRIX
             if ( mpStatusAttributes != NULL )
 #endif
                 mpAttributes = XVaAddToNestedList( mpAttributes, 
