--- /projects/sise/openoffice/devel/workpits/OOo641C/workarea/berkeleydb/makefile.mk	Sat Dec  1 01:42:25 2001
+++ berkeleydb/makefile.mk	Fri Jan 18 16:22:23 2002
@@ -79,7 +79,7 @@
 .IF "$(GUI)"=="UNX"
 CONFIGURE_DIR=out
 #relative to CONFIGURE_DIR
-CONFIGURE_ACTION=..$/dist$/configure
+CONFIGURE_ACTION=bash ..$/dist$/configure
 CONFIGURE_FLAGS=--enable-cxx --enable-java --enable-dynamic --enable-shared
 
 BUILD_DIR=$(CONFIGURE_DIR)
