Index: qa/qatesttool/global/system/inc/sysinfo.inc
===================================================================
RCS file: /cvs/qa/qatesttool/global/system/inc/sysinfo.inc,v
retrieving revision 1.2.2.1
diff -u -r1.2.2.1 sysinfo.inc
--- qa/qatesttool/global/system/inc/sysinfo.inc	27 Apr 2004 11:11:37 -0000	1.2.2.1
+++ qa/qatesttool/global/system/inc/sysinfo.inc	13 Oct 2005 06:03:28 -0000
@@ -96,6 +96,7 @@
 Global Const SYS_AIX   = 15
 Global Const SYS_IRI   = 16
 Global Const SYS_HP    = 17
+Global Const SYS_FBSD  = 18
 
 Global PlatFormGroup(MAX_PLATFORM) as String
 Global PlatFormName(MAX_PLATFORM) as String
@@ -120,6 +121,7 @@
 '///+<TR><TD>AIX</TD><TD>aix</TD><TD>unx</TD></TR>
 '///+<TR><TD>Irix</TD><TD>iri</TD><TD>unx</TD></TR>
 '///+<TR><TD>HP UNIX</TD><TD>hp</TD><TD>unx</TD></TR>
+'///+<TR><TD>FreeBSD</TD><TD>fbsd</TD><TD>unx</TD></TR>
 '///+</TABLE>
    PlatFormName(SYS_00)    = "Unknown"        : PlatFormExt(SYS_00)   = "a00"
    PlatFormName(SYS_W95)   = "Win95"          : PlatFormExt(SYS_W95)  = "w95"
@@ -137,6 +139,7 @@
    PlatFormName(SYS_AIX)   = "AIX"            : PlatFormExt(SYS_AIX)  = "aix"
    PlatFormName(SYS_IRI)   = "IRIX"           : PlatFormExt(SYS_IRI)  = "iri"
    PlatFormName(SYS_HP)    = "HP UNIX"        : PlatFormExt(SYS_Hp)   = "hp"
+   PlatFormName(SYS_FBSD)  = "FreeBSD"        : PlatFormExt(SYS_FBSD)   = "fbsd"
 
    PlatFormGroup(SYS_00)  = "a00"
    PlatFormGroup(SYS_W95)  = "w95"
@@ -154,6 +157,7 @@
    PlatFormGroup(SYS_AIX)  = "unx"
    PlatFormGroup(SYS_IRI)  = "unx"
    PlatFormGroup(SYS_HP)   = "unx"
+   PlatFormGroup(SYS_FBSD) = "unx"
 end sub
 
 '-------------------------------------------------------------------------
@@ -217,6 +221,7 @@
 '///+<TR><TD>AIX</TD><TD>04</TD></TR>
 '///+<TR><TD>Irix</TD><TD>06</TD></TR>
 '///+<TR><TD>HP UNIX</TD><TD>07</TD></TR>
+'///+<TR><TD>FreeBSD</TD><TD>08</TD></TR>
 '///+</TABLE>
 '///Get the path to <i>testtool.ini/.testtoolrc</i>
    if GUI = 4 then
@@ -295,6 +300,7 @@
             case 04 :   gSystemnummer = SYS_AIX
             case 06 :   gSystemnummer = SYS_IRI
             case 07 :   gSystemnummer = SYS_HP
+            case 08 :   gSystemnummer = SYS_FBSD
             case else:  Print "Unknown OS ("+GUI+"/"+iGuiVersion+") please insert in sysinfo.inc::GetToolPlatform. TestTool will try to use the same environment as Solaris SPARC uses."
                         gSystemnummer = SYS_SOL
          end select
@@ -354,6 +360,7 @@
          case "AIX"     : GUI = 15
          case "IRI"     : GUI = 16
          case "HP"      : GUI = 17        
+         case "FBSD"    : GUI = 18        
          case else      : print "Unknown operating system : '" + GuiString + "'. The should be canceled."
                           exit sub
       end select
Index: qa/qatesttool/script/unix/OOoTestRun_unix.sh
===================================================================
RCS file: /cvs/qa/qatesttool/script/unix/OOoTestRun_unix.sh,v
retrieving revision 1.1
diff -u -r1.1 OOoTestRun_unix.sh
--- qa/qatesttool/script/unix/OOoTestRun_unix.sh	9 Sep 2003 17:29:19 -0000	1.1
+++ qa/qatesttool/script/unix/OOoTestRun_unix.sh	13 Oct 2005 06:03:29 -0000
@@ -68,11 +68,11 @@
 
 # set location of testscripts
 # (the directory, where directory 'qatesttool' exists)
-sLocation=/testtool
+sLocation=/work/ooo/1.1.5/qa/
 
 # set location of TestTool
 # (full path including executable 'testtool')
-sTestTool=/testtool/linux/testtool
+sTestTool=/work/ooo/qatesttool/linux/testtool
 
 # set location of close-office file
 # (see cvs)
