--- /projects/sise/openoffice/devel/workpits/641-irix-gcc-STLP/source/transex3/source/localize.cxx	Tue Oct 16 02:20:40 2001
+++ transex3/source/localize.cxx	Wed Nov  7 09:20:49 2001
@@ -236,7 +236,7 @@
 	const ByteString &rParameter, const ByteString &rIso )
 /*****************************************************************************/
 {
-	if (( rIso == "noiso" ) || sIsoCode99.Len()) {
+	if (( rIso.Equals("noiso") ) || sIsoCode99.Len()) {
 		String sFull( rFileName, RTL_TEXTENCODING_ASCII_US );
 		DirEntry aEntry( sFull );
 		ByteString sFileName( aEntry.GetName(), RTL_TEXTENCODING_ASCII_US );
@@ -504,7 +504,7 @@
 	ByteString sExtension( aEntry.GetExtension(), RTL_TEXTENCODING_ASCII_US );
 	ByteString sCandidate( ExeTable[ nIndex ][ 0 ] );
 
-	while( !sCandidate.Equals ("NULL") && sCandidate != sExtension ) 
+	while( !sCandidate.Equals ("NULL") && !sCandidate.Equals(sExtension) ) 
 		sCandidate = ExeTable[ ++nIndex ][ 0 ];
 
 	ByteString sIso( ExeTable[ nIndex ][ 4 ] );
@@ -589,7 +589,7 @@
 		sFileName = sLine.GetToken( 0, '\t' );
 		sFileName += "#";
 		sFileName += sLine.GetToken( 1, '\t' );
-		if ( sFileName.Len() && ( sOldFileName != sFileName )) {
+		if ( sFileName.Len() && ( !sOldFileName.Equals(sFileName) )) {
 			if ( aFile.IsOpen()) {
 				aFile.Close();
 
