--- /projects/sise/openoffice/devel/workpits/OO641B_STLP45/source/sw/util/makefile.mk	Tue Oct  2 23:01:03 2001
+++ sw/util/makefile.mk	Fri Dec 14 16:14:03 2001
@@ -175,6 +175,8 @@
 
 SHL2STDLIBS= \
 	$(SALLIB) \
+	$(SDLIB) \
+	$(SCLIB) \
 	$(SFXLIB) \
 	$(OFALIB) \
 	$(SFXDEBUGLIB) \
@@ -197,8 +199,6 @@
 	$(UNOTOOLSLIB) \
 	$(VOSLIB) \
 	$(TKLIB) \
-	$(SDLIB) \
-	$(SCLIB) \
 	$(UCBHELPERLIB) \
 	$(XMLOFFLIB) \
 	$(COMPHELPERLIB)
@@ -242,9 +242,11 @@
 .IF "$(OS)"!="LINUX"
 .IF "$(OS)"!="FREEBSD"
 .IF "$(OS)"!="NETBSD"
+.IF "$(OS)"!="IRIX"
 SHL2OBJS+= \
 			$(SLO)$/atrfrm.obj      \
 			$(SLO)$/fmtatr2.obj
+.ENDIF
 .ENDIF
 .ENDIF
 .ENDIF
