--- /projects/sise/openoffice/devel/workpits/source/OO641/dbaccess/source/core/dataaccess/datasource.cxx	Wed Oct 10 01:41:46 2001
+++ dbaccess/source/core/dataaccess/datasource.cxx	Fri Nov 16 14:35:37 2001
@@ -59,6 +59,8 @@
  *
  ************************************************************************/
 
+#include <algorithm>
+
 #ifndef _DBA_COREDATAACCESS_DATASOURCE_HXX_
 #include "datasource.hxx"
 #endif
